app/services/context_service.rb

```diff
@@ -23,6 +32,19 @@ def org_admin_context
     end
   end
 
+  def service_to_service_headers
+    {
+      'x-rh-rbac-psk'       => ENV['RBAC_PSK'],
+      'x-rh-rbac-account'   => account_number,
+      'x-rh-rbac-client-id' => 'approval'
+    }
+  end
+
+  def account_number
+    Insights::API::Common::Request.current&.tenant ||
```

Comment:
@bzwei Is this for testing? Because you would always have the Insights::API::Common::Request.current.tenant because we are storing the context with every request in the database so when we have to restart something we would first initialize the request and then call the controller code.

Reply:
yes. it is very useful if we need to debug through rails console

Reply:
@bzwei For Testing acts_as_tenant has a simple way to do this from their console
```
# This manually sets the current tenant for testing purposes. In your app this is handled by the gem.
ActsAsTenant.current_tenant = Account.find(3)
```